static/src/stylesheets/module/_discussion.scss

```diff
@@ -240,6 +240,9 @@ $avatarPadding: $gs-gutter / 2;
         position: absolute;
         top: 10px;
         right: -320px;
```

Comment:
This numbers should use sizes from our grid system. 320 is gs-span(4) etc so they are not just some random magic numbers.